src/mono/wasm/runtime/net6-legacy/hybrid-globalization.ts

```diff
@@ -84,6 +84,76 @@ export function pass_exception_details(ex: any, exceptionMessage: Int32Ptr){
     exceptionRoot.release();
 }
 
+export function mono_wasm_starts_with(exceptionMessage: Int32Ptr, culture: MonoStringRef, str1: number, str1Length: number, str2: number, str2Length: number, options: number): number{
+    const cultureRoot = mono_wasm_new_external_root<MonoString>(culture);
+    try{
+        const cultureName = conv_string_root(cultureRoot);
+        const sd = new StringDecoder();
```

Comment:
you can use instance exported as `string_decoder` it will be already pre-initialized.